src/connection.js

```diff
@@ -1059,6 +1159,22 @@ class Connection extends EventEmitter {
     });
   }
 
+  sendFedAuthResponsePacket() {
+    const accessTokenLen = Buffer.byteLength(this.fedAuthInfo.token.accessToken, 'ucs2');
+    const data = new Buffer(8 + accessTokenLen);
+    let offset = 0;
+    data.writeUInt32LE(accessTokenLen + 4, offset);
```

Comment:
`Buffer#writeUInt32LE` returns the new offset after writing to the buffer, so you could simplify this like:

```js
let offset = 0;
offset = data.writeUInt32LE(accessTokenLen + 4, offset);
offset = data.writeUInt32LE(accessTokenLen, offset);
data.write(this.fedAuthInfo.token.accessToken, offset, 'ucs2');
```